src/Component/CategoryDetail/Component/MdChoice/Pagination.js

```diff
@@ -4,28 +4,27 @@ import "./Pagination.scss";
 class Pagination extends Component {
   render() {
     const { fetchProduct, activePage } = this.props;
-    console.log(activePage);
 
     return (
       <div className="Pagination">
         <button
           data-idx="0"
           onClick={fetchProduct}
-          className={activePage === "0" ? "active" : null}
+          className={activePage === 0 ? "active" : null}
```

Comment:
```suggestion
          className={activePage === 0 ? "active" : ""}
```
null값이 나오면 클래스명이 null이 되기 때문에 빈 스트링을 주셔야 할것 같습니다